react-native/react-native.d.ts

```diff
@@ -3354,7 +3386,99 @@ declare namespace  __React {
     }
 
     export interface StyleSheetStatic extends React.ComponentClass<StyleSheetProperties> {
-        create<T>( styles: T ): T;
```

Comment:
Yes I was looking at this too an I am afraid changing this will break a lot of existing code.
I suggest improving the definition by fixing the return type which is currently wrong, i.e. updating it to

`create<T>( styles: T ): {[key: string]: number};`